pkg/ccl/backupccl/schedule_pts_chaining.go

```diff
@@ -312,11 +312,11 @@ func protectTimestampRecordForSchedule(
 	targetToProtect *ptpb.Target,
 	deprecatedSpansToProtect roachpb.Spans,
 	tsToProtect hlc.Timestamp,
-	scheduleID int64,
+	scheduleID jobspb.ScheduleID,
 ) (uuid.UUID, error) {
 	protectedtsID := uuid.MakeV4()
 	return protectedtsID, pts.Protect(ctx, jobsprotectedts.MakeRecord(
-		protectedtsID, scheduleID, tsToProtect, deprecatedSpansToProtect,
+		protectedtsID, int64(scheduleID), tsToProtect, deprecatedSpansToProtect,
```

Comment:
Not for this PR, but in a follow up perhaps we should change the API here so that we can take the correct type.

Reply:
Yeah, I think if we do this we just break this up into two different functions each of which just produces one kind of record rather than having the caller specify the type.